juniper_official/interface/check-ethernet-crc-framing-errors.rule

```diff
@@ -0,0 +1,217 @@
+/*
+ * Monitors Cyclic Redundancy,In-Fragment and Jabber frames errors and notifies when anomalies are found
```

Comment:
s/Cyclic Redundancy/Cyclic Redundancy check/g or CRC

Reply:
Changed to CRC